coss/opensource_clubs/templates/opensource_clubs/category_page.html

```diff
@@ -0,0 +1,7 @@
+{% extends "opensource_clubs/category_base.html" %}
```

Comment:
Is this new template going to break the existing site in prod?

Reply:
No, it's expected to have no effect on existing templates. We just change the template all these pages inherit from, but the hierarchy remains the same.